include/seqan3/alignment/matrix/detail/score_matrix_single_column.hpp

```diff
@@ -92,7 +93,8 @@ class score_matrix_single_column
      * Note the alignment matrix requires the number of columns and rows to be one bigger than the size of sequence1,
      * respectively sequence2.
      * Reallocation happens only if the new column size exceeds the current capacity of the optimal and horizontal
-     * score column. The underlying vectors will not be cleared during the reset.
+     * score column. The underlying vectors are initialised with the given `initial_value` or the default value of
+     * the class' score type.
```

Comment:
I think the possesive form would be correct here since we are talking about the class its score type. But I checked and in singluar possesive form ending with an s you write `class's` and spell it class-es.